invenio_rdm_records/resources/serializers/datacite/schema.py

```diff
@@ -76,18 +78,25 @@ def get_name_identifiers(self, obj):
         """Get name identifier list."""
         serialized_identifiers = []
         identifiers = obj["person_or_org"].get("identifiers", [])
-
+        scheme_uri = ""
         for identifier in identifiers:
             scheme = identifier["scheme"]
             id_scheme = get_scheme_datacite(
                 scheme, "RDM_RECORDS_PERSONORG_SCHEMES", default=scheme
             )
 
             if id_scheme:
+                if scheme == "orcid":
+                    scheme_uri = "http://orcid.org/"
+                elif scheme == "ror":
+                    scheme_uri = "https://ror.org/"
+
                 name_id = {
                     "nameIdentifier": identifier["identifier"],
                     "nameIdentifierScheme": id_scheme,
+                    "schemeUri": scheme_uri,
```

Comment:
`scheme_uri` can be `""`, there's no fallback. In that case, do we want to add the `schemeUri`  key in the serialized identifier?